pkg/insights/insightsclient/insightsclient.go

```diff
@@ -38,13 +37,11 @@ const (
 )
 
 type Client struct {
-	client      *http.Client
-	maxBytes    int64
-	metricsName string
-
-	authorizer       Authorizer
-	gatherKubeConfig *rest.Config
-	clusterVersion   *configv1.ClusterVersion
+	client             *http.Client
+	maxBytes           int64
+	metricsName        string
+	authorizer         Authorizer
+	gatherConfigClient *configv1client.ConfigV1Client
```

Comment:
Not sure why it is called `gatherConfigClient`. It doesn't have anything in common with data gathering IMO.

Reply:
Honestly I'm not sure either, I had probably something in mind about clusterversion being gathered but I agree this is a bad name.

Reply:
Just found it, I just kept the name that was already there, my bad.